app/actions/settings.jsx

```diff
@@ -17,3 +17,9 @@ export const saveSettings = createAction(
   ACTION_TYPES.SETTINGS_SAVE,
   data => data
 );
+
+// Notify user about invalid decimal fractions
+export const notifyInvalidDecimalFractions = createAction(
```

Comment:
Please use `dialog` for validation message. I would like to user to acknowledge this before moving on.

Reply:
Changed :)